config/routes.rb

```diff
@@ -12,8 +12,6 @@
   get '/api/openid_connect/userinfo' => 'openid_connect/user_info#show'
   post '/api/risc/security_events' => 'risc/security_events#create'
 
-  post '/api/address_search' => 'idv/in_person/public/address_search#index'
```

Comment:
@soniaconnolly 's comment at https://github.com/18F/identity-idp/pull/9136#discussion_r1319226674 reminded me here, is the removal of these paths going to cause some issues during deployment for users running JavaScript served by old boxes sending requests to new servers?

Reply:
No, the current status of the `in_person_full_address_entry_enabled` ensures that users will not be using this endpoint in production.